### PR TITLE
New. Integrations. Added honeypot for elementor form.

### PR DESCRIPTION
**Pull request by svfcode**
task https://doboard.com/1/task/9433

### CODE REVIEW

File: inc/cleantalk-public-integrations.php [hunk 1]
```diff
@@ -3324,7 +3262,11 @@ function apbct_form__elementor_pro__addField($content)
 {
     global $apbct;
 
-    if ( $apbct->settings['trusted_and_affiliate__under_forms'] === '1' && strpos($content, '</form>') !== false ) {
+    $search = '</form>';
+    $replace = ct_add_honeypot_field('elementor_form') . $search;
```
Comment (Glomberg) [inline inc/cleantalk-public-integrations.php:3266]: Please, check if the option `Add a honeypot field` is enabled.

File: inc/cleantalk-settings.php [hunk 2]
```diff
@@ -3306,7 +3306,7 @@ function apbct_settings__get__long_description()
             //HANDLE LINK
             'desc'  => sprintf(
                 esc_html__('The option helps to block bots . The honeypot field option adds a hidden field to the form. When spambots come to a website form, they can fill out each input field. Enable this option to make the protection stronger on these forms. Learn more about supported forms %s', 'cleantalk-spam-protect'),
-                '<a href="https://cleantalk.org/help/wordpress-plugin-settings{utm_mark}#honeypot" target="_blank">' . __('here.', 'cleantalk-spam-protect') . '</a>'
+                '<a href="https://cleantalk.org/help/honeypot-field{utm_mark}" target="_blank">' . __('here.', 'cleantalk-spam-protect') . '</a>'
```
Comment (Glomberg) [inline inc/cleantalk-settings.php:3309]: Is it possible to use link builder here?

**Review by Glomberg — CHANGES_REQUESTED** [review]

**Review by alexandergull — APPROVED** [review]
Approve, please, fix the issues above.

File: inc/cleantalk-public-integrations.php (hunk 1 above)
Comment (svfcode) [reply inc/cleantalk-public-integrations.php:3266]: Option checks inside ct_add_honeypot_field() https://github.com/CleanTalk/wordpress-antispam/blob/master/inc/cleantalk-public.php#L699

File: inc/cleantalk-settings.php (hunk 2 above)
Comment (svfcode) [reply inc/cleantalk-settings.php:3309]: Yeah, it is possible, but requires refactoring here https://github.com/CleanTalk/wordpress-antispam/blob/master/inc/cleantalk-settings.php#L3367
If you think it is needed, then will be continue of task.

**Pull request merged**